build.sbt

```diff
@@ -16,7 +16,7 @@ val assertNoApplicationConf = taskKey[Unit]("Makes sure application.conf isn't p
 val commonSettings = Seq(
   organization := "com.spingo",
   version := appProperties.getProperty("version"),
-  scalaVersion := "2.12.10",
+  scalaVersion := "2.11.8",
```

Comment:
2.11 ? :)

Reply:
Yes, reverted to the original  `scalaVersion`.

Reply:
Can you check once again please?